pkgs/applications/misc/pass-secret-service/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+with stdenv.lib;
+with python3.pkgs;
+
+buildPythonApplication rec {
+  pname = "pass_secret_service";
+  # Official release in PyPI is old, let's use latest from GitHub
```

Comment:
Old is not necessarily bad. Does it miss important bug and security fixes or does it not work with the current versions of other software in nixpkgs?

Reply:
Yes, I agree. I looked into this a bit more carefully now: The release in PyPI seems to me like a semi-random snapshot of the project and its version number is 0.1a0 - so, not a real release anyway, but alpha. Since then the project has switched from using a seemingly abandoned D-Bus package `pydbus` and started using maintained `dbus-next`. So, in my opinion, this unstable version makes perfect sense. What do you think? If so, I could write this more detailed reasoning in that comment.

Reply:
I now added a bit more details why it's an unstable version.